// @fluentui/react-northstar/dist/commonjs/themes/teams-forced-colors/components/SplitButton/splitButtonToggleStyles.js
"use strict";

exports.__esModule = true;
exports.splitButtonToggleStyles = void 0;
var _toggleIndicatorUrl = require("../../../teams/components/SplitButton/toggleIndicatorUrl");
var getIndicatorStyles = function getIndicatorStyles(color, outline, size) {
  return {
    content: '""',
    width: size,
    height: size,
    backgroundImage: (0, _toggleIndicatorUrl.toggleIndicatorUrl)(color, outline),
    backgroundRepeat: 'no-repeat'
  };
};
var splitButtonToggleStyles = {
  root: function root(_ref) {
    var p = _ref.props,
      v = _ref.variables,
      theme = _ref.theme;
    var siteVariables = theme.siteVariables;
    var borderWidth = siteVariables.borderWidth;
    return Object.assign({
      ':before': Object.assign({}, getIndicatorStyles(p.disabled ? 'GrayText' : 'ButtonText', true, v.toggleButtonIndicatorSize))
    }, p.primary && {
      borderWidth: borderWidth,
      borderColor: "transparent"
    }, p.disabled && {
      borderWidth: borderWidth,
      borderColor: "transparent"
    });
  }
};
exports.splitButtonToggleStyles = splitButtonToggleStyles;
